src/picongpu/include/simulation_defines/unitless/speciesAttributes.unitless

```diff
@@ -30,9 +28,32 @@
 #include "particles/traits/MacroWeighted.hpp"
 #include "particles/traits/WeightingPower.hpp"
 
+
+/** \file speciesAttributes.unitless
+ *
+ * This file describes the particle attributes defined in
+ * \see speciesAttributes.param of a particle species.
+ * Each particle attribute needs to implement the following traits
+ *   - `Unit`
+ *      - conversion between PIConGPU units and SI
+ *      - \see traits/Unit.hpp
+ *      - \see https://git.io/vriio
+ *   - `UnitDimension`
+ *      - powers of the seven SI base units
+ *      - \see traits/UnitDimension.hpp
+ *      - \see https://git.io/vriio
```

Comment:
I think, this should point to [one paragraph lower](https://github.com/openPMD/openPMD-standard/blob/1.0.0/STANDARD.md#required-for-each-record). Not to [https://git.io/vriio](https://git.io/vriio).